android/source/VideoLocker/src/org/edx/mobile/logger/LogUtil.java

```diff
@@ -0,0 +1,74 @@
+package org.edx.mobile.logger;
+
+import android.content.Context;
+import android.content.pm.ApplicationInfo;
+import android.util.Log;
+
+
+class LogUtil {
+
+    public static boolean ISDEBUG = true;
+
+    /**
+     * Initializes <code>LogUtil.ISDEBUG</code> 
+     * @param context - current context passed
+     */
+    public static void init(Context context) {
+        boolean isDebuggable =  ( 0 != ( context.getApplicationInfo().flags & ApplicationInfo.FLAG_DEBUGGABLE));
```

Comment:
@shahidtamboli To be more specific, this line should use <code>ApplicationInfo.FLAG_DEBUGGABLE ==</code> instead of <code>0 !=</code>.